### ISSUE TITLE
Report Fuzz Introspector findings to project contacts

### ISSUE THREAD

**Issue by cgzones** [google/oss-fuzz#12616]
The Fuzz Introspector reported (correctly) that the [checkpolicy-fuzzer of the selinux project is broken](https://storage.googleapis.com/oss-fuzz-introspector/selinux/inspector-report/20241016/fuzz_report.html).
Maybe such findings can be automatically send via email to the project contacts.
Also the report (and a diff to the previous one to detect stalls) can be send e.g. in a quarterly interval.

**Comment by evverx** [comment]
I think it's kind of related to https://github.com/google/oss-fuzz/issues/11398.

**Comment by DavidKorczynski** [comment]
I agree, and also think this is related to https://github.com/google/oss-fuzz/issues/11398

I like the idea a lot of sending out quarterly emails with a summary of the state/progress/issues/etc.

I'll try to get the ball moving on this one.

**Comment by phi-go** [comment]
@DavidKorczynski is this a policy question or does this just need an implementation?

If it just requires implementation and if you can give me pointer where this would be best added I can take a look at implementing this.
